portal/templates/redesign/teach_new/class_new.html

```diff
@@ -100,8 +126,10 @@ <h4 class="col-sm-6 col-center">Here you can view and manage all of your student
     <p class="body-text">Here you may change the name of the class, transfer the class to another teacher in the school or club, create permissions for the class to
         accept requests from external students, or delete classes for which there are no students. You can also create permissions for students in this class to see
         their classmates&rsquo; progress.</p>
-    <button class="button--regular button--primary--positive">Edit settings</button>
-    <button class="button--regular button--primary--danger">Delete class</button>
+    <div class="background button-group">
+        <a id="class_settings_button" href="{% url 'teacher_edit_class_new' class.access_code %}" class="button--regular button--primary--positive">Edit settings</a>
+        <button id='deleteClass' class="button--regular button--primary--danger">Delete class</button>
```

Comment:
Be as consistent about your ID naming convention as you are about your classes, or everything will get really difficult to follow, really fast.